pilot/pkg/serviceregistry/kube/controller/ambientindex.go

```diff
@@ -225,6 +225,9 @@ func (a *AmbientIndex) matchesScope(scope model.WaypointScope, w *model.Workload
 }
 
 func (c *Controller) Policies(requested sets.Set[model.ConfigKey]) []*workloadapi.Authorization {
+	if isNil(c.configController) {
+		return []*workloadapi.Authorization{}
+	}
```

Comment:
Actually in this case, we need to get authz policy from config cluster. Otherwise the logic is not  right

Reply:
This is called from a loop over all sources - it seems that remote clusters only pull endpoints. I'm not entirely sure how configController ends up null for remote clusters - all I did was add the secrets. I can debug - but it seems correct to not have a config controller for multi cluster remotes.

If MeshConfig.configSources is used I assume we'll have multiple sources providing authz - and the logic will work. I saw this pattern in another place in this file too.